examples/discovery-search-app/cypress/tsconfig.json

```diff
@@ -4,7 +4,7 @@
     "baseUrl": "../node_modules",
     "target": "es5",
     "lib": ["es5", "dom"],
-    "types": ["cypress"]
+    "types": ["cypress", "jest"]
```

Comment:
what does `jest` have to do with cypress?

Reply:
I think I added this to get rid of an error message about missing types for jest assertions within the cypress tests (such as in autocomplete/spec.ts). But I removed this now, and nothing has broken so far... Maybe whatever was throwing the error also got cleaned out in my initial pruning.